.template/variants/web/.eslintrc

```diff
@@ -4,5 +4,8 @@
   ],
   "globals": {
     "I18n": true
+  },
+  "parserOptions":{
+    "ecmaVersion": "latest"
```

Comment:
`eslint-config-nimble-core` has [parserOptions](https://github.com/nimblehq/eslint-config-nimble/blob/e63c501e54498a5319276b2a828bb94fa3665f06/packages/eslint-config-nimble-core/lib/index.js#L8-L9)

Could you check whether it solves this issue i.e changing Line number 3 to "@nimblehq/eslint-config-nimble/core"




btw we already have a issue to change `"@nimblehq/eslint-config-nimble"` to `"@nimblehq/eslint-config-nimble-core"`: https://github.com/nimblehq/rails-templates/issues/422, you might close two issues with one PR 😄

Reply:
Hello @tyrro ,

I tried to follow your suggestions.
just replacing `"@nimblehq/eslint-config-nimble-core"` under `.eslintrc` receive 
```
ESLint couldn't find the config "@nimblehq/eslint-config-nimble-core"
```

So I tried to run yarn add `@nimblehq/eslint-config-nimble-core --dev` and change in the `.eslintrc` and tried to run 
 `yarn eslint`  and receive the following errors.
```
Oops! Something went wrong! :(

ESLint: 8.44.0

Error: Failed to load plugin '@typescript-eslint' declared in '.eslintrc » @nimblehq/eslint-config-nimble-core » /Users/htooaeinlwin/work/hello/node_modules/@nimblehq/eslint-config-nimble-core/lib/rules/typescript.js': Cannot find module 'typescript'
Require stack:
- /Users/htooaeinlwin/work/hello/node_modules/@typescript-eslint/eslint-plugin/dist/util/astUtils.js
```

Reply:
@tyrro the `@nimblehq/eslint-config-nimble-core` is not ready for the Rails project yet, see [this issue](https://github.com/nimblehq/eslint-config-nimble/issues/43) and [this issue](https://github.com/nimblehq/eslint-config-nimble/issues/56) so we are still with the `eslint-config-nimble` and that requires this config :)